src/templates/finops-hub/modules/dataFactory.bicep

```
@@ -5048,9 +5048,9 @@ resource pipeline_ExecuteIngestionETL 'Microsoft.DataFactory/factories/pipelines
           ]
         }
       }
-      {
+          {
         name: 'Data Explorer validation'
-        description: 'if a Data Explorer instance is deployed, the status will be checked and if the instance is in state "Stopped" the instance will be started'
+        description: 'when the FTK is deployed with an Azure Data Explorer instance, a start of the instance will be initiated to ensure a running instance'
```

nit: Removing the "FTK" reference and simplifying. This is for FinOps hubs specifically. The toolkit doesn't get "deployed" today. It's just a container.
```suggestion
        description: 'If Data Explorer is stopped, start it'
```